Tools/TestWebKitAPI/wpe/PlatformWebViewWPE.cpp

```diff
@@ -67,13 +71,29 @@ PlatformWebView::PlatformWebView(WKPageRef relatedPage)
 PlatformWebView::~PlatformWebView()
 {
     WKRelease(m_view);
-    delete m_window;
+#if ENABLE(WPE_PLATFORM)
+    if (g_type_class_peek(WPE_TYPE_DISPLAY))
```

Comment:
That is sufficient to detect if the _current_ runtime chosen API is the new platform API?